server/services/PopulatedData/populateData.js

```diff
@@ -24,7 +24,12 @@ const getTests = require('../../models/services.deprecated/TestsService');
  * and no database queries will be run by this function.
  * @returns
  */
-const populateData = async (locationOfData, { preloaded } = {}) => {
+const populateData = async (
+    locationOfData,
+    { transaction, preloaded } = {}
```

Comment:
It is a required field. It can't be defaulted because that would create a divergent database state which would differ with the rest of the request.

In the case that the transaction is undefined all the model methods will raise an error like this:

```
Please provide a transaction via the "transaction" field or pass a value of false to specify that a transaction is not needed.
```

Reply:
Got it!